docs/core/utilities/logging.md

```diff
@@ -0,0 +1,129 @@
+  ACTS_LOCAL_LOGGER(myLogger);
+  ACTS_VERBOSE("This message will not appear in the logfile.");
+  ACTS_INFO("But this one will: Hello World!");
+  // do not forget to close the logfile
+  logfile.close();
+}
+```
+
+## Logger integration
+
+In case you are using Acts in another framework which comes with its own
+logging facility (e.g. Gaudi) you can pipe the logging output from Acts
+tools and algorithms to your framework's logging system by supplying different
+implementations of:
+
+- {class}`Acts::Logging::OutputFilterPolicy` (for mapping logging levels)
+- {class}`Acts::Logging::OutputPrintPolicy` (for passing the Acts output
+  to your internal logging system)
+
+There are two approaches to logger integration:
+
+1. [](override_deflog).
+   This has the downside that log levels cannot be controlled from top-level
+   experiment specific code. This means that it is non-trivial to steer the log
+   level of an e.g. Gaudi algorithm via the `OutputLevel` property, and have
+   the ACTS code respect this log level. It is therefore now **discouraged** to
+   use this approach.
+
+    :::{note}
+    ACTS code has iteratively moved to not construct loggers via `getDefaultLogger`
+    as much as possible, in favor of using {class}`Acts::LoggerWrapper`. The latter
+    is a lightweight non-owning wrapper around a logger instance, that can default
+    to not containing a logger. It is more suitable to pass into functions that
+    might be called from other ACTS functions (rather than construction a local
+    logger via `getDefaultLogger`). 
+    :::
+
+2. Passing logger instances to high level components, and rely on ACTS code to
+   pass them into lower level classes / functions.
+
+
+(override_deflog)=
+### Overriding `Acts::getDefaultLogger()`
+
+:::{attention}
+Using this mechanism is now **discouraged** for integration with an experiment
+framework.
+:::
+
+Since Acts makes extensive use of {func}`Acts::getDefaultLogger()` to provide
+sufficient information for debugging, you might want to provide a modified
+implementation of this function (using your output filter and printing
+policies) to also pipe this output to your framework. You can use the following
+approach using the possibility to inject custom code by pre-loading shared
+libraries with `LD_PRELOAD`. You need to provide an appropriate implementation
+for a function of the following signature into a separate source file and
+compile it in a shared library
+
+
+```cpp
+namespace Acts {
+std::unique_ptr<const Logger> getDefaultLogger(const std::string&,
+                                               const Logging::Level&,
+                                               std::ostream*);
+}
+```
+
+Then you can run your executable, which uses Acts tools and algorithms, in
+the following way (tested under Unix)
+
+```console
+$ LD_PRELOAD=<YOUR_SHARED_LIBRARY> path/to/your/exectuable
+```
+
+For an example have a look at CustomDefaultLogger.cpp which you can use as
```

Comment:
I think \`CustomDefaultLogger.cpp\` is missing the grave accents.

Reply:
```suggestion
For an example have a look at `CustomDefaultLogger.cpp` which you can use as
```